pages/linux/mono.md

```diff
@@ -0,0 +1,20 @@
+# mono
+
+> Mono, the open source development platform based on the .NET Framework.
+> More information: <https://www.mono-project.com/docs/>.
+
+- Compile:
+
+`csc {{source_file}}.cs`
+
+- Run:
+
+`mono {{source_file}}.exe`
+
+- Compile using a specific library:
```

Comment:
Again, I'm not sure this example belongs here, since we already have a page for `csc`.